src/components/docker/ContainerRow.tsx

```
@@ -218,24 +220,24 @@ export default memo(function ContainerRow({ container, chartData }: ContainerRow
-              to="/docker/$containerId"
-              params={{ containerId: container.id.split('/')[1] }}
-              search={{ host: container.id.split('/')[0], metrics: 'cpu,memory' }}
-              onClick={(e: React.MouseEvent) => e.stopPropagation()}
+            <button
+              onClick={(e) => {
+                e.stopPropagation();
+                onOpenHistory?.(container.id.split('/')[1], container.id.split('/')[0]);
+              }}
               className="p-1 rounded opacity-0 group-hover:opacity-100 transition-opacity hover:bg-neutral-500/20"
               aria-label="View container history"
             >
```

Biome (2.4.4)

[error] Provide an explicit type prop for the button element.

(lint/a11y/useButtonType)